### PR TITLE
[TGP-423] Implement Update Record endpoint

### PR DESCRIPTION
**Pull request by pauloncast**
Implemented Update Record endpoint

### CODE REVIEW

**Review by william-eyre — APPROVED** [review]

File: app/uk/gov/hmrc/tradergoodsprofiles/models/responses/CreateOrUpdateRecordResponse.scala [hunk 1]
```diff
@@ -74,18 +72,16 @@ object CreateRecordResponse {
         (json \ "toReview").as[Boolean],
         (json \ "reviewReason").asOpt[String],
         (json \ "declarable").as[String],
-        (json \ "ukimsNumber").as[String],
-        (json \ "nirmsNumber").as[String],
-        (json \ "niphlNumber").as[String],
-        (json \ "locked").as[Boolean],
-        (json \ "srcSystemName").as[String],
```
Comment (AmarkumarJagtap) [inline app/uk/gov/hmrc/tradergoodsprofiles/models/responses/CreateOrUpdateRecordResponse.scala:81]: do we not need these fields?

Comment (paulonevrything) [reply app/uk/gov/hmrc/tradergoodsprofiles/models/responses/CreateOrUpdateRecordResponse.scala:81]: `locked` and `srcSystemName` are not part of the spec response for create and update. They are only returned as part of get records

Comment (pauloncast) [reply app/uk/gov/hmrc/tradergoodsprofiles/models/responses/CreateOrUpdateRecordResponse.scala:81]: `locked` and `srcSystemName` are not part of the spec response for create and update. They are only returned as part of get records

**Review by AmarkumarJagtap — APPROVED** [review]

**Pull request merged**